docker/build-tools/Dockerfile

```
@@ -27,9 +27,9 @@
 
 FROM golang:1.13.4 as binary_tools_context
 
-# Istio tools branch that we use for this image
-ARG ISTIO_TOOLS_BRANCH
-ENV ISTIO_TOOLS_BRANCH=${ISTIO_TOOLS_BRANCH}
+# Istio tools SHA that we use for this image
+ARG ISTIO_TOOLS_SHA
```

```suggestion
ARG ISTIO_TOOLS_SHA=master
```

So you can just run `docker build` and things work

---

This will not work for non-master branch.